meeting-notes/20181127.md

```diff
@@ -0,0 +1,227 @@
+# Dynamic Data and Capabilities WG — November 27th, 2018
+
+- **Lead:** @pgte
+- **Notetaker:** @jimpick
+- **Attendees:**
+  - @pgte
+  - @jimpick
+  - @andreforsousa
+  - @daviddahl
+  - @arkadiy
+  - @aschmahmann
+  - @marcooliveira
+  - @satazor
+
+- [**Zoom.us meeting URL**](https://protocol.zoom.us/j/586782711)
+- [**Recording**](https://www.youtube.com/watch?v=CY_cVPEATc8&feature=youtu.be)
+
+## Agenda
+
+1. Start recording
+1. Ask everyone to put their name into the list of attendees
+1. Ask for a volunteer to take notes
+1. Everyone can add items to this agenda for things they would like to discuss
+1. Round of intros and updates
+  - Introduce yourself and your interest in this subject
+  - What have you accomplished since the last meeting?
+  - Were there any blockers? If so, which ones? Is it still blocked? Why?
+  - What is the next important thing you should focus on?
+1. Demos
+  - peer-pad E2E load tests
+  - discussify replies
+1. Questions
+
+### 2 Week Updates:
+
+(please copy this and add your update here :)
+
+#### @
+ - Concluded:
+ - In progress:
+ - Blocked:
+ - Next:
+
+#### @satazor
+ - Concluded:
+   - Replies
+   - Lazy load of comments and replies (view more)
+ - In progress:
+   - versidag + ipfs-versidag modules
+     - https://github.com/ipfs/dynamic-data-and-capabilities/issues/50
```

Comment:
@satazor check out these threads:

- https://github.com/ipfs/notes/issues/23
- https://github.com/ipfs/notes/issues/45
- https://github.com/ipfs/notes/issues/195

They are full of insights that will be useful for you :)